easy-ui-storybook/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@easypost/easy-ui-storybook",
```

Comment:
how do we feel about this being named `-storybook` instead of a more abstract name like`-docs`, etc?